packages/relay/src/lib/config/hbarSpendingPlanConfigService.ts

```diff
@@ -108,16 +109,19 @@ export class HbarSpendingPlanConfigService {
    * @private
    */
   private loadSpendingPlansConfig(): SpendingPlanConfig[] {
-    const configPath = findConfig(this.SPENDING_PLANS_CONFIG_FILE);
-    if (!configPath || !fs.existsSync(configPath)) {
-      this.logger.trace(`Configuration file not found at path "${configPath ?? this.SPENDING_PLANS_CONFIG_FILE}"`);
-      return [];
+    const configPath = findConfig(this.SPENDING_PLANS_CONFIG);
+    if (!this.SPENDING_PLANS_CONFIG) {
```

Comment:
Q: shouldn't this be the returned value since the value is extracted for you already 
```suggestion
    if (!this.spendingPlanConfig) {
```

Reply:
+1, we should check the result from `findConfig` here, not the constant that holds the env variable.

Reply:
Both the `spendingPlanConfig` and `findConfig` we also used here.